Frameworks/UIKit/WYPopoverController.m

```diff
@@ -1459,10 +1463,15 @@ - (CGRect)innerRect:(CGRect)rect arrowDirection:(WYPopoverArrowDirection)aArrowD
         result.size.height -= borderWidth;
     }
     
+    // The below code is disabled on WINOBJC due to some issue where |viewContentInsets| has a garbage value.
+    // Given that most of the theming/drawing code in WYPopoverController is effectively a no-op in WINOBJC,
+    // just skipping over the below is an adequate (or at least quick) fix.
```

Comment:
@jaredhms the best/only fix for the issue is still the current pull. 

Trying the following instead of the current pull still leaves the bug:
```
diff --git a/Frameworks/UIKit/WYPopoverController.m b/Frameworks/UIKit/WYPopoverController.m
index 442df3c..89b0bd2 100644
--- a/Frameworks/UIKit/WYPopoverController.m
+++ b/Frameworks/UIKit/WYPopoverController.m
@@ -606,7 +606,7 @@ static char const * const UINavigationControllerEmbedInPopoverTagKey = "UINaviga
 }

 - (NSArray *)observableKeypaths {
-    return [NSArray arrayWithObjects:@"tintColor", @"outerStrokeColor", @"innerStrokeColor", @"fillTopColor", @"fillBottomColor", @"glossShadowColor", @"glossShadowOffset", @"glossShadowBlurRadius", @"borderWidth", @"arrowBase", @"arrowHeight", @"outerShadowColor", @"outerShadowBlurRadius", @"outerShadowOffset", @"outerCornerRadius", @"innerShadowColor", @"innerShadowBlurRadius", @"innerShadowOffset", @"innerCornerRadius", @"viewContentInsets", @"overlayColor", nil];
+    return [NSArray arrayWithObjects:@"tintColor", @"outerStrokeColor", @"innerStrokeColor", @"fillTopColor", @"fillBottomColor", @"glossShadowColor", @"glossShadowOffset", @"glossShadowBlurRadius", @"borderWidth", @"arrowBase", @"arrowHeight", @"outerShadowColor", @"outerShadowBlurRadius", @"outerShadowOffset", @"outerCornerRadius", @"innerShadowColor", @"innerShadowBlurRadius", @"innerShadowOffset", @"innerCornerRadius", /*@"viewContentInsets",*/ @"overlayColor", nil];
 }

 @end
@@ -983,6 +983,8 @@ static float edgeSizeFromCornerRadius(float cornerRadius) {
         self.arrowOffset = 0;

         self.layer.name = @"parent";
+
+        self.viewContentInsets = UIEdgeInsetsZero;

         if (WY_IS_IOS_GREATER_THAN_OR_EQUAL_TO(@"6.0"))
         {

```